flytekit/extend/__init__.py

```diff
@@ -1,4 +1,34 @@
-from flytekit.common.tasks.sdk_runnable import ExecutionParameters
+"""
+=====================
+Extending Flytekit
+=====================
+
+.. currentmodule:: flytekit.extend
+
+This package contains things that are useful when extending Flytekit.
+
+.. autosummary::
+   :toctree: generated/
+
+   get_serializable - fdjsalkfsd
+   context_manager - naive
+   SQLTask
+   IgnoreOutputs
+   PythonTask
+   ExecutionState
+   Image
+   ImageConfig
+   SerializationSettings
+   Interface
+   Promise
+   TaskPlugins
+   DictTransformer
+   T
```

Comment:
?

Reply:
I have no idea what this is.  But one of the plugins uses it!  I've yet to figure it out.